src/components/file-upload.tsx

```diff
@@ -134,6 +134,7 @@ export class GenerateTable extends React.Component<GenerateTableProps> {
       var earningsYears = Object.keys(parsedXml);
       tableRows = earningsYears.map((year, i) => {
         const earningValueXML = parsedXml[year]
+        /* Can this be parseable with symbols $, */
```

Comment:
See below
```suggestion
```

Reply:
Looks good otherwise! I can commit this suggestion and get this merged in if you agree.

Reply:
Thanks Thad, that works for me. In the future I'll just capture something like this in notes and/or eventually an issue.